packages/paste-website/src/pages/index.tsx

```diff
@@ -8,10 +8,11 @@ import {Box} from '@twilio-paste/box';
 import {Anchor} from '@twilio-paste/anchor';
 import {Heading} from '@twilio-paste/heading';
 import {Paragraph} from '@twilio-paste/paragraph';
+import {UnorderedList, ListItem} from '@twilio-paste/list';
 import {SiteWrapper} from '../components/site-wrapper';
 import {Grid} from '../components/grid';
 import {StyledHr} from '../components/StyledHr';
-import {P, UL, LI} from '../components/Typography';
+import {P} from '../components/Typography';
```

Comment:
Do we still need this `P` since we're importing `Paragraph` above?

Reply:
Yeah the website has this concept of a lead paragrapgh, which is bigger than the rest. I could replace it with just `Text` but it "works" as is so thought I'd just leave it.